src/components/ui/nice-select/index.tsx

```diff
@@ -30,7 +34,10 @@ const NiceSelect = ({
   useClickAway(ref, onClose);
 
   const currentHandler = (item: Option) => {
-    setCurrent(item);
+    setCurrent({
```

Comment:
You are writing extra code with this approach

Reply:
hmm I don't have anything in my mind to change it. Do you have any suggestions?